src/haxeLanguageServer/features/haxe/GotoDefinitionFeature.hx

```diff
@@ -10,8 +10,10 @@ import languageServerProtocol.Types.DefinitionLink;
 class GotoDefinitionFeature {
 	final context:Context;
 
-	public function new(context) {
+	public function new(context, listen = true) {
```

Comment:
whem using `context.gotoDefinition` there should be no need for a private instance of `GotoDefinitionFeature`, so `listen` is unnecessary (see comment in `createMissingArgumentsAction`)